internal/metrics/command.go

```diff
@@ -2,21 +2,31 @@ package metrics
 
 import (
 	"context"
-	"time"
 	"github.com/spf13/cobra"
+	"github.com/superfly/flyctl/internal/instrument"
+	"sync"
+	"time"
 )
 
 var (
 	processStartTime = time.Now()
-	commandContext context.Context
+	mu               sync.Mutex
+	commandContext   context.Context
 )
 
-type commandTimingData struct {
-	Duration float64 `json:"duration_seconds"`
-	Command  string  `json:"command"`
+type commandStats struct {
+	Command         string  `json:"c"`
+	Duration        float64 `json:"d"`
+	GraphQLCalls    int     `json:"gc"`
+	GraphQLDuration float64 `json:"gd"`
+	FlapsCalls      int     `json:"fc"`
+	FlapsDuration   float64 `json:"fd"`
 }
 
 func RecordCommandContext(ctx context.Context) {
+	mu.Lock()
+	defer mu.Unlock()
```

Comment:
unrelated to this PR, but found we were missing mutex lock here. Unlikely to cause real problems, but strictly speaking, it is necessary.